api/proto/teleport/legacy/types/types.proto

```diff
@@ -1254,7 +1254,18 @@ message ProvisionTokenSpecV2 {
     (gogoproto.casttype) = "Duration"
   ];
   // JoinMethod is the joining method required in order to use this token.
-  // Supported joining methods include "token", "ec2", and "iam".
+  // Supported joining methods include:
```

Comment:
As Zac said, this will not render nicely:
<img width="914" alt="image" src="https://github.com/user-attachments/assets/5e593e85-c42e-4466-9e3d-12c8f1c20b7c">

We could improve the doc generator, but at this point I think it's better to just list the options separated by a comma.

Reply:
Yeah - I'll add another commit in a minute to resolve this.